src/Events/Notifications/Notifications.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Class that handles interfacing with TEC\Common\Notifications.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Notifications
+ */
+
+namespace TEC\Events\Notifications;
+
+use Tribe__Events__Main as TEC;
+
+/**
+ * Class Notifications
+ *
+ * @since   TBD
+ * @package TEC\Events\Notifications
+ */
+class Notifications {
+
+	/**
+	 * Determines if we are on a TEC admin page except the post edit page.
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
+	 */
+	public static function is_tec_admin_page(): bool {
```

Comment:
We have a  couple versions of this, we need to consolidate them and add hooks as needed, rather than re-creating another version. See: `src/Events/Telemetry/Telemetry.php` for an example.